_topic_map.yml

```diff
@@ -2680,8 +2680,8 @@ Topics:
       File: virt-dedicated-resources-vm
     - Name: Scheduling virtual machines
       File: virt-schedule-vms
-    - Name: Configuring PCI passthrough
-      File: virt-configuring-pci-passthrough
```

Comment:
do you intentionally drop virt-configuring-pci-passthrough ?

Reply:
Nope - not sure what happened there. Will take a look.